### DISCUSSION TITLE
application run after installing typer-cli

### DISCUSSION THREAD

**Discussion by petre-renware** [tiangolo/typer-cli#147] [Questions] [answered]
... I do not know what exactly is missing... Is something related to <default> ref an argument parameter. It has a default defined as documentation (classig parameter with default val, using = atfer its list in function parameters). Running the application with `typer` command also does not work, but is less important because the app need and is becoming Widows EXE, which withoout `typer-cli` runs perfectly and everybody is happy with. I installed typer-cli just to generate the application documentation as markdon file to be included in main siet of the system (made with mkdocs), but I solve for this version by running application with --help for all options, collecting the resulted output and consolidate it in a md file. It would be nice if typer-cli could be used and if possible to fix the problem of requiring the default positional parameter ONLY when typer-clie is installed... Thanks. A very nice and beautiful day for everybody !

**Comment by sixdimensionalarray** [comment]
I ran into this also.  installing `typer-cli` downgrades your `typer` version to 0.7.0 if you aren't careful.  And that version (0.7.0) has this problem.

Please see: https://github.com/tiangolo/typer-cli/issues/125

I was running `typer` == 0.9.0 prior to installing the `typer-cli`.  To fix the issue I did:
```bash
pip uninstall typer-cli
pip install --upgrade typer
```
This puts back typer 0.9.0.

Then, you can get the typer-cli back via:

```bash
pip install --no-deps typer-cli
```
and run the doc generation and it should probably work.

That said, that's a pretty yucky workaround, so I don't recommend it.

**Comment by tiangolo** [comment, marked as answer]
Hello! Now Typer-CLI is part of Typer, since version 0.12.1, so now you can just install `pip install typer` and you get the `typer` command. :rocket: 

You can try with the latest version and see if that solves it.

If not, please create a new question following all the steps, including a simple example I can copy-paste to debug your use case: https://github.com/tiangolo/typer/discussions/new?category=questions
